openshift/build-tool-detector.app.yaml

```diff
@@ -0,0 +1,121 @@
+kind: Template
+apiVersion: v1
+metadata:
+  name: build-tool-detector
+objects:
+- kind: DeploymentConfig
+  apiVersion: v1
+  metadata:
+    labels:
+      service: build-tool-detector
+    name: build-tool-detector
+  spec:
+    replicas: ${{REPLICAS}}
+    selector:
+      service: build-tool-detector
+    strategy:
+      rollingParams:
+        intervalSeconds: 1
+        maxSurge: 25%
+        maxUnavailable: 25%
+        timeoutSeconds: 600
+        updatePeriodSeconds: 1
+      type: Rolling
+    template:
+      metadata:
+        labels:
+          service: build-tool-detector
+      spec:
+        containers:
+        - image: ${IMAGE}:${IMAGE_TAG}
+          env:
+          - name: BUILD_TOOL_DETECTOR_GITHUB_CLIENT_ID
+            valueFrom:
+              secretKeyRef:
+                name: build-tool-detector
+                key: github.client.id
+          - name: BUILD_TOOL_DETECTOR_GITHUB_CLIENT_SECRET
+            valueFrom:
+              secretKeyRef:
+                name: build-tool-detector
+                key: github.client.secret
+          - name: BUILD_TOOL_DETECTOR_SENTRY_DSN
+            valueFrom:
+              secretKeyRef:
+                name: build-tool-detector
+                key: github.sentry.dsn
+          - name: BUILD_TOOL_DETECTOR_AUTH_URI
+            valueFrom:
+              configMapKeyRef:
+                name: build-tool-detector
+                key: auth.uri
+          imagePullPolicy: Always
+          name: build-tool-detector
+          ports:
+          - containerPort: 8099
+            protocol: TCP
+          livenessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /api/status
+              port: 8099
```

Comment:
Is this still the correct port? I think I saw a commit go in to expose 8089? https://github.com/fabric8-services/build-tool-detector/commit/062999fe2a0e3fb0307d8a2c89b104d035eb699b

Reply:
I actually moved everything over to 8099 in a newer commit since you had started with 8099 :)

Reply:
Ohhhh :-) haha okay, sounds good!